tests/resources/VCH-Util.robot

```diff
@@ -489,7 +489,8 @@ Cleanup Datastore On Test Server
     \   ${state}=  Get State Of Drone Build  @{build}[1]
     \   Continue For Loop If  '${state}' == 'running'
     \   Log To Console  Removing the following item from datastore: ${item}
-    \   ${out}=  Run  govc datastore.rm ${item}
+    \   ${rc}  ${out}=    Run And Return Rc And Output  govc datastore.rm ${item}
```

Comment:
If something can not be delete, may it lead to problems with other tests? And if item cannot be delete, what's the possible reason except network hickups and similar?

Reply:
couldn't be deleted because there was a vm attached to it still, other tests cleanup vm's first before trying to delete datastore so wouldn't see this problem.  it only happens on this test because this is specifically testing the vic-service delete functionality in this way.